website/content/en/v0.31/concepts/_index.md

```diff
@@ -129,6 +129,10 @@ When Karpenter detects one of these events will occur to your nodes, it automati
 Karpenter publishes Kubernetes events to the node for all events listed above in addition to __Spot Rebalance Recommendations__. Karpenter does not currently support cordon, drain, and terminate logic for Spot Rebalance Recommendations.
 {{% /alert %}}
 
+{{% alert title="Note" color="warning" %}}
+For users that wish to benefit from both Karpetner and Spot Rebalance Recommendations, which usually entails using the AWS Node Termination Handler, please note that it may cause a disruption between how karpenter handles Spot instances and their events and how AWS Node Termination Handler does, furter information can be found here: [Troubleshooting](../troubleshooting.md#Node-provisioning-and-deprovisioning-loop)
```

Comment:
Added some spelling fixes and some hyperlinks. Let me know if these suggestions make sense:
```suggestion
For users that wish to benefit from both Karpenter and [Spot Rebalance Recommendations](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/rebalance-recommendations.html), which usually entails using the [AWS Node Termination Handler](https://github.com/aws/aws-node-termination-handler), please note that the AWS Node Termination Handler cordons and drains nodes, potentially causing node churn in the cluster. Further information can be found here: [Troubleshooting](../troubleshooting.md#Node-provisioning-and-deprovisioning-loop)
```